modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationMetricsListener.java

```diff
@@ -121,11 +134,17 @@ public TcpCommunicationMetricsListener(GridMetricManager mmgr) {
         );
 
         sentMsgsCntByNodeIdMetricFactory = nodeId ->
-            mmgr.registry(metricName(COMMUNICATION_METRICS_GROUP_NAME, nodeId.toString()))
-                .findMetric(SENT_MESSAGES_BY_NODE_ID_METRIC_NAME);
+            allSentMsgsMetricsByNodeId.computeIfAbsent(nodeId, k -> new AtomicLong());
 
         rcvdMsgsCntByNodeIdMetricFactory = nodeId ->
-            mmgr.registry(metricName(COMMUNICATION_METRICS_GROUP_NAME, nodeId.toString()))
+            allRcvdMsgsMetricsByNodeId.computeIfAbsent(nodeId, k -> new AtomicLong());
+
+        sentMsgsCntByConsistentIdMetricFactory = consistentId ->
```

Comment:
Metric named `ConsistentIdMetric` but the constant for looking up named as by nodeId `SENT_MESSAGES_BY_NODE_ID_METRIC_NAME` why?

Reply:
I'll rename it.